boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi

```diff
@@ -63,6 +83,13 @@
 
```

Comment:
The pmic_iset needs to be here and set low.

Reply:
this is already handled by the PMIC driver.

Reply:
Checked: OK